defaults/main.yml

```diff
@@ -75,8 +75,8 @@ openio_event_agent_filter_content_rebuild:
 openio_event_agent_filter_content_cleaner:
   use: "egg:oio#content_cleaner"
 
-openio_event_agent_filter_meta2_index:
-  use: "egg:oio#meta2_index"
+openio_event_agent_filter_volume_index:
+  use: "egg:oio#volume_index"
```

Comment:
`volume_index` is already defined in this file, we can just delete these 2 lines.